src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/SelectionPropertyMetadataMapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\SchemaMetadata;
+
+use Sulu\Component\Content\Metadata\PropertyMetadata as ContentPropertyMetadata;
+
+class SelectionPropertyMetadataMapper implements PropertyMetadataMapperInterface
+{
+    public function mapPropertyMetadata(ContentPropertyMetadata $propertyMetadata): PropertyMetadata
+    {
+        $mandatory = $propertyMetadata->isRequired();
+        $minItems = $propertyMetadata->getParameter('min', true)['value'] ?? null;
+        $maxItems = $propertyMetadata->getParameter('max', true)['value'] ?? null;
```

Comment:
i think this is a nice feature! should we call it something like `maxItems` or `maxSelections`? we also need to document it once this is merged 😇

Reply:
@alexander-schranz suggested `min` and `max` and I also think that's a good idea, because then we can use `min` and `max` also for other multi-value-properties like blocks ... if we would use `minSelections`, we could probably only use that param for selections, because it's name doesn't fit for blocks

Reply:
i think thats the advantage of using params here - we dont need to use the same param name for different things. 
for example, we could use descriptive names such as `minItemCount` for selections and `minBlockCount` for blocks. but if everybody else thinks we should try to reuse param names, thats okay for me too 🙂

Reply:
I think it would be easier for developers, if they don't have to remember lots of different param names for different properties, if their purpose is the same

Reply:
I also think that `min` and `max` are precise enough in the context of the content type, so I don't think I would give them any longer names.